spec/services/intake_progress_calculator_spec.rb

```diff
@@ -10,23 +10,13 @@
     end
 
     it "returns 100 for the progress of the final controller and intake" do
-      progress = IntakeProgressCalculator.get_progress(Questions::AdditionalInfoController, intake)
+      progress = IntakeProgressCalculator.get_progress(Questions::SuccessfullySubmittedController, intake)
 
       expect(progress).to eq 100
     end
 
     it "returns the same values for the DependentsController and the Questions:HadDependentsController" do
       expect(IntakeProgressCalculator.get_progress(DependentsController, intake)).to eq IntakeProgressCalculator.get_progress(Questions::HadDependentsController, intake)
     end
-
-    it "includes all possible future steps until you reach the question" do
-      ever_married_yes = Intake.new(ever_married: :yes)
-      ever_married_no = Intake.new(ever_married: :no)
-      ever_married_unfilled = Intake.new
-      controller_before_question = Questions::IssuedIdentityPinController
-
-      expect(IntakeProgressCalculator.get_progress(controller_before_question, ever_married_yes)).to eq IntakeProgressCalculator.get_progress(controller_before_question, ever_married_no)
-      expect(IntakeProgressCalculator.get_progress(controller_before_question, ever_married_yes)).to eq IntakeProgressCalculator.get_progress(controller_before_question, ever_married_unfilled)
-    end
```

Comment:
this is related to the behavior change I commented about in app/services/intake_progress_calculator.rb above. i don't see why this is desirable behavior, but i may be missing something.